src/main/java/edu/harvard/iq/dataverse/UserNotificationServiceBean.java

```diff
@@ -103,6 +103,10 @@ public void sendNotification(AuthenticatedUser dataverseUser, Timestamp sendDate
     }
 
     public void sendNotification(AuthenticatedUser dataverseUser, Timestamp sendDate, Type type, Long objectId, String comment, AuthenticatedUser requestor, boolean isHtmlContent) {
+        if (dataverseUser.isDisabled()) {
```

Comment:
I don't think we need this: disabled users will have no roles. Let's save a few lines of code.

Reply:
Agreed. I was just being paranoid, I guess. Removed in 8f08f7b31.